controllers/projectclaimadapter.go

```diff
@@ -87,9 +84,9 @@ func (c *ProjectClaimAdapter) ProjectReferenceExists() (bool, error) {
 }
 
 // EnsureProjectClaimDeletionProcessed deletes ProjectClaim in cases a deletion was triggered
-func (adapter *ProjectClaimAdapter) EnsureProjectClaimDeletionProcessed() (gcputil.OperationResult, error) {
-	if adapter.IsProjectClaimDeletion() {
-		crState, err := adapter.FinalizeProjectClaim()
+func (c *ProjectClaimAdapter) EnsureProjectClaimDeletionProcessed() (gcputil.OperationResult, error) {
```

Comment:
this is not supposed to be c, but p.
nitpicking as CAD has a linter that checks this for us (I think `revive`)

Reply:
I've kept it `c` for now.